### PR TITLE
fix: Add handling of unknown type attachment.

### PR DESCRIPTION
**Pull request by kutuzov13**
# Checklist

- Fix crash bot if type attachment is not support.

- [x] I've written good commit message for all commits
- [ ] I've split changes into separate commits where it's appropriate
- [ ] I've updated project version in `pyproject.toml`
- [ ] I'll make a release when PR is merged

### CODE REVIEW

File: botx/models/message/incoming_message.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import logging
```
Comment (lxmnk) [inline botx/models/message/incoming_message.py:1]: `from botx.logger import logger`

File: botx/models/message/incoming_message.py [hunk 2]
```diff
@@ -223,22 +224,28 @@ def to_domain(self, raw_command: Dict[str, Any]) -> IncomingMessage:  # noqa: WP
...
-            if isinstance(attachment_domain, FileAttachmentBase):
-                file = attachment_domain
-            elif attachment_domain.type == AttachmentTypes.LOCATION:
-                location = attachment_domain
-            elif attachment_domain.type == AttachmentTypes.CONTACT:
-                contact = attachment_domain
-            elif attachment_domain.type == AttachmentTypes.LINK:
-                link = attachment_domain
+            if isinstance(self.attachments[0], dict):
+                logging.warning("Unsupported attachment type")
```
Comment (lxmnk) [inline botx/models/message/incoming_message.py:234]: `logger.warning("Received unsupported attachment type")`

File: tests/test_attachments.py [hunk 3]
```diff
@@ -270,3 +270,48 @@ async def default_handler(message: IncomingMessage, bot: Bot) -> None:
...
+
+
+API_UNSUPPORTED_FILE_ATTACHMENT_TYPE = (
+    {
+        "data": {
+            "id": "9657694f-166e-575a-b6ea-6575521d4f57",
+            "pack": "2c00198f-6efb-5b36-bed1-1a380bc85710",
+            "version": 1637674405,
+        },
+        "type": "sticker",
```
Comment (lxmnk) [inline tests/test_attachments.py:282]: If we add stickers, this test will fail. Use
```json
{
  "data": {"foo": "bar"},
  "type": "baz"
}
```

File: tests/test_attachments.py [hunk 4]
```diff
@@ -270,3 +270,48 @@ async def default_handler(message: IncomingMessage, bot: Bot) -> None:
...
+
+    collector = HandlerCollector()
+    incoming_message: Optional[IncomingMessage] = None
+
+    @collector.default_message_handler
+    async def default_handler(message: IncomingMessage, bot: Bot) -> None:
+        nonlocal incoming_message
+        incoming_message = message
+        # Drop `raw_command` from asserting
+        incoming_message.raw_command = None
```
Comment (lxmnk) [inline tests/test_attachments.py:308]: You can drop this lines, 'cause you don't asserting `incoming_message`.

File: tests/test_attachments.py [hunk 5]
```diff
@@ -270,3 +270,48 @@ async def default_handler(message: IncomingMessage, bot: Bot) -> None:
...
+        "type": "sticker",
+    },
+)
+
+
+@pytest.mark.parametrize(
+    "api_attachment",
+    API_UNSUPPORTED_FILE_ATTACHMENT_TYPE,
+)
+async def test__async_execute_raw_bot_command__unsupported_attachment_types(
```
Comment (lxmnk) [inline tests/test_attachments.py:291]: `unsupported_attachment_type`

File: tests/test_attachments.py [hunk 6]
```diff
@@ -270,3 +270,48 @@ async def default_handler(message: IncomingMessage, bot: Bot) -> None:
...
+            "id": "9657694f-166e-575a-b6ea-6575521d4f57",
+            "pack": "2c00198f-6efb-5b36-bed1-1a380bc85710",
+            "version": 1637674405,
+        },
+        "type": "sticker",
+    },
+)
+
+
+@pytest.mark.parametrize(
```
Comment (lxmnk) [inline tests/test_attachments.py:287]: You don't need parametrize here, you use only one attachment for test.

File: botx/models/message/incoming_message.py (hunk 2 above)
Comment (kutuzov13) [reply botx/models/message/incoming_message.py:234]: Fix.

File: tests/test_attachments.py (hunk 4 above)
Comment (kutuzov13) [reply tests/test_attachments.py:308]: Fix.

File: tests/test_attachments.py (hunk 5 above)
Comment (kutuzov13) [reply tests/test_attachments.py:291]: Fix.

File: tests/test_attachments.py (hunk 6 above)
Comment (kutuzov13) [reply tests/test_attachments.py:287]: Fix.

File: tests/test_attachments.py (hunk 3 above)
Comment (kutuzov13) [reply tests/test_attachments.py]: Fix.

**Review by lxmnk — APPROVED** [review]

**Review by Nimond — APPROVED** [review]

**Pull request merged**